screenshot-testing/build.gradle

```diff
@@ -0,0 +1,39 @@
+apply from: configs.androidLibrary
+
+dependencies {
+    implementation project(':stripe-ui-core')
+    implementation libs.compose.ui
+    implementation libs.compose.foundation
+    implementation libs.compose.material
+
+    // Implemented here instead of using plugin which implements with testImplementation
+    implementation("app.cash.paparazzi:paparazzi:${versions.paparazzi}") {
+        exclude group: 'org.hamcrest'
```

Comment:
How come these are needed?

Reply:
To resolve duplicate class errors. I think they weren't raised for the plugin because it is a testImplementation but are raised for api/implementation

Reply:
If you add an explicit dependency on `testLibs.hamcrest` does the conflict go away?

Reply:
It does not. However, I did add these incrementally as I was resolving errors. I retested which ones needed to be excluded and it's only `'jakarta.activation'`,  `'com.android.tools.ddms', module: 'ddmlib'`, and `'layoutlib-native-jdk11'` so I removed the others

Reply:
Cool. Thanks for digging!